app/ocr.py

```diff
@@ -134,7 +142,31 @@ def similar_outcome(str1, str2):
 
     return False
 
-# TODO: This static list should be stored and accessed via the backend 
+"""
+LISTS
+
+global info about judges; states and their court circuits
+"""
+
+circuit_dict = {
+    '''used by get_circuit'''
+    'DC': 'DC', 'ME': '1', 'MA': '1', 'VT': '1', 'RI': '1', 'PR': '1',
+    'CT': '2', 'NY': '2', 'VT': '2', 'DE': '3', 'PA': '3', 'NJ': '3', 'VI': '3',
+    'MD': '4', 'VA': '4', 'NC': '4', 'SC': '4', 'WV': '4', 'LA': '5', 'MS': '5',
+    'TX': '5', 'KY': '6', 'OH': '6', 'TN': '6', 'MI': '6', 'IL': '7', 'IN': '7',
+    'WI': '7', 'AK': '8', 'IA': '8', 'MN': '8', 'MO': '8', 'NE': '8', 'ND': '8',
+    'SD': '8', 'AK': '9', 'AZ': '9', 'CA': '9', 'GU': '9', 'HI': '9', 'ID': '9',
+    'MT': '9', 'NV': '9', 'MP': '9', 'OR': '9', 'WA': '9', 'CO': '10',
+    'KS': '10', 'NM': '10', 'OK': '10', 'UT': '10', 'WY': '10', 'AL': '11',
+    'FL': '11', 'GA': '11'
+ }

```

Comment:
Most of this is in judges_courts.csv, we can add state abbreviations to it if we need them, but either way it's better to load in the file globally in whatever format we want and avoid hardcoding any of this information.

Reply:
+1 I would appreciate this data in a json file for creating visualizations.
For context, we are trying to create a choropleth map using the districts, but the web back-end currently only sends us state information. Adding the above data to a separate file would be a quick fix to that problem.